packages/editor/src/components/post-visibility/style.scss

```diff
@@ -0,0 +1,19 @@
+.editor-post-visibility {
+	.editor-post-visibility__dialog-legend {
+		font-weight: 600;
+		margin-bottom: 1em;
+		margin-top: 0.5em;
+	}
+	.editor-post-visibility__choice {
+		label {
+			font-weight: 600;
+		}
+		input {
+			margin-right: 0.5em !important;
```

Comment:
Minor: Can we try to avoid the usage of import here ( e.g: adding a new class to the input and if necessary repeat it)?